scss/button/_layout.scss

```diff
@@ -177,6 +177,21 @@
 
     }
 
+    .k-responsive-button-group {
+        display: flex;
+
+        > .k-button {
+            display: inline-block;
+            flex: 1 1 0;
+            overflow: hidden;
+            text-overflow: ellipsis;
+
+            > .k-icon {
+                vertical-align: text-bottom;
```

Comment:
Is this required only for this type of button group? It appears like something that should be applied to all button groups.